.github/scripts/compile.sh

```diff
@@ -19,13 +19,15 @@
 # under the License.
 #
 
-set -euxo pipefail
+# https://sipb.mit.edu/doc/safe-shell
+set -Eefuxo pipefail
 
-WORKING=$(pwd)
-eval "$(rpmbuild --eval '%set_build_flags')"
+WORKING="$(pwd)"
+BUILD_FLAGS="$(rpmbuild --eval '%set_build_flags')"
+eval "${BUILD_FLAGS}"
```

Comment:
Splitting this into assignment and `eval` will fail the assignment if `rpmbuild` does not exist.